src/integrationTest/java/uk/gov/hmcts/divorce/systemupdate/event/SystemUpdateCaseWithCourtHearingIT.java

```diff
@@ -298,6 +305,8 @@ public void givenBothApplicantsAreRepresentedAndIsPaperApplicationThenNoEmailsSe
             .getResponse()
             .getContentAsString();
 
+        verify(bulkPrintService, times(2)).print(any());
```

Comment:
are we sending request twice to bulk print or is it two different letters ?

Reply:
It is two different letters